### ISSUE TITLE
Accessibility feedback - Staging Review - BAM2 [Debt Letters MVP]

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12166]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Jennifer (VSA) & Trevor (VSP)

## Description

This ticket houses accessibility issues for the staging review of BAM2 - Debt Letters MVP.

## Test cases

1. Staging link: https://staging.va.gov/manage-va-debt/your-debt/ and please use Greg Anderson (user 1). This tests a user with a number of debts and debt letters. 
1. CMS preview of DL langing-page: http://preview-prod.vfs.va.gov/preview?nodeId=6521

### PDF information for the VA 508 Office

https://github.com/department-of-veterans-affairs/va.gov-team/issues/11692#issuecomment-683095595

> Here are the VBMS contacts for you :

> VBMS PM – Clyde Miller – clyde.miller@va.gov

> VBMS Intake Manager – Michael Pohlman – Michael.pohlman@va.gov

## Definition of done

1. Review and acknowledge feedback.
1. Document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions.

**Comment by jenstrickland** [comment]
### Tickets will be documented, notes taken here from the VA environment.


**CMS landing page**
1. The heading level issue is with the Option 1: Pay online, Option 2: Pay by phone, and Option 3: Pay by mail. Each of these are `h4`, but there isn't a preceding `h3`. These should be `h3`s.

![Screen Shot 2020-08-12 at 1 19 21 PM](https://user-images.githubusercontent.com/57469/90046445-7b4feb00-dc9e-11ea-8895-ef6f481c1c7f.png)

![Screen Shot 2020-08-12 at 1 21 05 PM](https://user-images.githubusercontent.com/57469/90046729-dc77be80-dc9e-11ea-8397-582cd1d4bb29.png)

2. The aside issue in in the CMS template, documented in https://github.com/department-of-veterans-affairs/va.gov-team/issues/7844 and https://github.com/department-of-veterans-affairs/va.gov-team/issues/6379


**Also**, the hub link and text issue is on this page.

![Screen Shot 2020-08-12 at 1 26 36 PM](https://user-images.githubusercontent.com/57469/90047221-8ce5c280-dc9f-11ea-8ed2-5f772d9dcdee.png)

**Comment by jenstrickland** [comment]
@rebeccawalsh @rileyorr While conducting the accessibility staging review, I noticed this instruction to "please go to Pay your VA copay bill" and that this isn't linked. Is it possible to provide a link to this? When listening to this on a screen reader I was unsure where I may find that, and it would be especially challenging for low-vision and blind users. 

![Screen Shot 2020-09-14 at 6 09 16 PM](https://user-images.githubusercontent.com/57469/93143198-72937000-f6b5-11ea-923f-c047dbe882c1.png)

**Comment by jenstrickland** [comment]
@1Copenut — This accessibility staging review is ready for your review. The CMS landing page was minimally reviewed, due to lack of tools in the VA environment, and will have a full audit during the Full Accessibility & 508 Office Audit. 

cc @leahkeeler FYSA

**Comment by rileyorr** [comment]
Hi @jenstrickland - great catch! There's meant to be a link in the sentence about medical co-payments. It should read:

> For medical co-payment debt, please go to [Pay your VA copay bill](https://www.va.gov/health-care/pay-copay-bill/) to learn about your payment options.

**Comment by 1Copenut** [comment]
@jenstrickland I've reviewed your issue tickets and they look good. I added a couple of small comments to one issue, but otherwise no additional feedback.

**Comment by jenstrickland** [comment]
@leahkeeler — this is complete and ready for your team's review.
